src/js/_enqueues/admin/common.js

```diff
@@ -2245,3 +2245,105 @@ $( function( $ ) {
 	// Expose public methods.
 	return pub;
 })();
+
+/**
+ * Initializes tooltips for elements with the class 'tooltip-container'.
+ *
+ * Dynamically creates tooltip elements and attaches event listeners.
+ * Ensures tooltips are initialized only once.
+ *
+ * @since 6.5.0
+ */
+jQuery(function ($) {
+    var tooltipsInitialized = false;
+
+    /**
+     * Initializes tooltips for elements with the class 'tooltip-container'.
+     *
+     * @since 6.5.0
+     */
+    function initializeTooltips() {
+        // Select all tooltip containers on the page
```

Comment:
```suggestion
        // Select all tooltip containers on the page.
```

Missing end period.

🔢 applies to all inline doc.